apps/omni/src/assets/token_info.json

```diff
@@ -97,5 +97,16 @@
     "extensions": {
       "coingeckoId": ""
     }
+  },
+  {
+    "address": "J1toso1uCk3RLmjorhTtrVwY9HJ7X8V9yYac6Y7kGCPn",
+    "chainId": 101,
+    "decimals": 9,
+    "name": "Jito Staked SOL",
+    "symbol": "JitoSOL",
+    "logoURI": "https://storage.googleapis.com/token-metadata/JitoSOL-256.png",
```

Comment:
I think you need to whitelist that address in the next config file for this to work?

Reply:
Will switch to local one